plugins/Compressor/Compressor.cpp

```diff
@@ -177,7 +177,8 @@ void CompressorEffect::calcRange()
 
 void CompressorEffect::resizeRMS()
 {
-	m_rmsTimeConst = exp(-1.f / (m_compressorControls.m_rmsModel.value() * 0.001f * m_sampleRate));
+	const float rmsValue = m_compressorControls.m_rmsModel.value();
+	m_rmsTimeConst = (rmsValue != 0) ? exp(-1.f / (rmsValue * 0.001f * m_sampleRate)) : 0;
```

Comment:
It's generally advised to avoid strict equality comparisons with floating point numbers. In this case, negative numbers are undesirable too, so you could replace `!=` with `>`.